todoman/cli.py

```diff
@@ -282,21 +307,19 @@ def edit(ctx, id, todo_properties, interactive):
 @cli.command()
 @pass_ctx
 @with_id_arg
+@catch_errors
```

Comment:
Yeah, leave it as-is for now, I've some ideas for this and `pass_ctx`, but they're out of scope for this.